assets/docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+x-logging: &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "4"
+
+services:
+  proxy:
+    image: nginx:1-alpine
+    ports:
+      # Listen for outside connections on port 10633. You can freely change the left-side
+      # number to a different port, eg using port 80 if you don't need a reverse proxy.
+      - "10633:8536"
+    volumes:
+      - ./nginx_internal.conf:/etc/nginx/nginx.conf:ro,Z
+      - ./proxy_params:/etc/nginx/proxy_params:ro,Z
+    restart: always
+    logging: *default-logging
+    depends_on:
+      - pictrs
+      - lemmy-ui
+
+  lemmy:
+    image: dessalines/lemmy:0.19.5
```

Comment:
We're gonna need to remember to up these manually.

Reply:
I think we had some docs for the release process somewhere, but mosty ignore it. Maybe echo a message about upgrading these in the backend release script?